{"uuid": "3ce3bcfe-8828-4e9d-ba6e-5d5d503d2091", "children": ["64c0c073-4a6f-42b2-b7fd-6494a0448995"], "befores": [{"name": "add_user_setup_down", "status": "broken", "statusDetails": {"message": "AttributeError: 'SubRequest' object has no attribute 'param'\n", "trace": "  File \"D:\\Python\\Python38-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Python\\Python38-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 937, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Python\\Python38-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 790, in call_fixture_func\n    res = next(it)\n  File \"E:\\python\\vmc-interface\\vmc-interface\\testcases\\conftest.py\", line 49, in add_user_setup_down\n    parameter = request.param\n"}, "start": 1720167851577, "stop": 1720167851577}], "start": 1720167851577, "stop": 1720167851981}